rosa_hcp/rosa-hcp-egress-lockdown-install.adoc

```diff
@@ -51,28 +51,61 @@ While you may install and upgrade your clusters as you would a regular cluster,
 ====
 
 [id="rosa-hcp-egress-lockdown-install-creating_{context}"]
-== Creating a Virtual Private Cloud for your egress lockdown {hcp-title} clusters
+== Creating a Virtual Private Cloud for your {hcp-title} clusters
 
-You must have a Virtual Private Cloud (VPC) to create {hcp-title} clusters. You can use one of the following methods to create a VPC:
+You must have a Virtual Private Cloud (VPC) to create {hcp-title} cluster. Use one of the following methods to create a VPC:
 
+* Create a VPC using the ROSA command-line interface (CLI)
 * Create a VPC by using a Terraform template
 * Manually create the VPC resources in the AWS console
 
 [NOTE]
 ====
-The Terraform instructions are for testing and demonstration purposes. Your own installation requires modifications to the VPC for your specific needs and constraints. You should also ensure that when you use the following Terraform script it is in the same region that you intend to install your cluster.
+The Terraform instructions are for testing and demonstration purposes. Your own installation requires some modifications to the VPC for your own use. You should also ensure that when you use this Terraform script it is in the same region that you intend to install your cluster. In these examples, use `us-east-2`.
 ====
 
-include::modules/rosa-hcp-vpc-terraform.adoc[leveloffset=+2]
+[discrete]
+include::modules/rosa-hcp-create-network.adoc[leveloffset=+3]
+[discrete]
+include::modules/rosa-hcp-custom-create-network.adoc[leveloffset=+3]
 
 [role="_additional-resources"]
+[id="additional-resources_rosa-hcp-create-network-egress-lockdown"]
 .Additional resources
 
-* See the link:https://github.com/openshift-cs/terraform-vpc-example/tree/main/zero-egress[Zero Egress Terraform VPC Example] repository for a detailed list of all options available when customizing the VPC for your needs.
+* See the link:https://aws.amazon.com/cloudformation/[AWS CloudFormation] for more information about structuring CloudFormation files to create VPCs.
```

Comment:
`[AWS CloudFormation documentation]`? (Also, I believe current standards are to use only the link in the bullet, not provide additional text.)